src/views/Send/Send.vue

```diff
@@ -351,9 +357,23 @@ export default {
     ...mapState(['activeNetwork', 'activeWalletId', 'fees', 'fiatRates']),
     ...mapGetters('app', ['ledgerBridgeReady']),
     ...mapGetters(['accountItem', 'client', 'suggestedFeePrices']),
+    nativeAsset() {
+      return this.getNativeAsset(this.asset)
+    },
+    accountUrl() {
+      return `/accounts/${this.accountId}/${this.nativeAsset}/receive`
+    },
     account() {
       return this.accountItem(this.accountId)
     },
+    nativeAssetBalance() {
+      const balance = this.account.balances?.[this.nativeAsset] || 0
+      const pBalance = prettyBalance(balance, this.nativeAsset)
```

Comment:
@mpwanyi256 if you will need to convert the balance to BN, maybe we don't need to call `prettyBalance` just use the balance directly, `prettyBalance` is just to format and set the decimal places

Reply:
@bradleySuira Oh.. okay